flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendTestBase.java

```diff
@@ -82,13 +90,19 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
+import com.esotericsoftware.kryo.serializers.JavaSerializer;
```

Comment:
Can we use `o.a.f.api.java.typeutils.kryo.JavaSerializer` instead (see https://issues.apache.org/jira/browse/FLINK-6025)? 
I don't think it really matters for the tests, but we probably should start using it instead of Kryo's because of the bug in case anything unexpected pops up.